services/121-service/test/scope/indirect-relation-config-exhaustiveness.test.ts

```diff
@@ -0,0 +1,23 @@
+import { HttpStatus } from '@nestjs/common';
+
+import { SeedScript } from '@121-service/src/scripts/enum/seed-script.enum';
+import { getServer } from '@121-service/test/helpers/utility.helper';
+import { resetDB } from '@121-service/test/helpers/utility.helper';
+
+describe('Development Controller - Indirect Relations Validation', () => {
+  beforeEach(async () => {
+    await resetDB(SeedScript.nlrcMultiple, __filename);
+  });
+
+  // This test checks if indirectRelationConfig (in scoped.repository.ts) is exhaustive, as it is hard to maintain this manually over time.
+  // It is done via API-test instead of unit-test, because it was not possible to it requires the database to be set up.
+  // If this test fails, then you can find more details on missing details in the 121-service logs, which can be used to update the indirectRelationConfig.
```

Comment:
Now that I read this it would be more convenient if the endpoint return the error message in the body. 

Than you can compare it here in this test file with an empty string so you can conveniently read it from CI/CD if it goes wrong.

Reply:
done